.travis.yml

```diff
@@ -1,10 +1,19 @@
-dist: xenial
+matrix:
+  include:
+    - dist: xenial
```

Comment:
Can we name these two jobs? Right now the names are "python" and "node.js"
```
matrix:
  include:
    - name: "RST check"
       dist: xenial
...
    - name: "YAML/JSON check"
       dist: trusty
```
https://docs.travis-ci.com/user/customizing-the-build#naming-jobs-within-matrices

Reply:
Done.